service AudioService {
  string audio_conversion(1:binary audio)

}
